### PR TITLE
Add Crash Stat uptake task.

### PR DESCRIPTION
**Pull request by Natim**
- [X] Update `API_CHANGELOG.rst`
- [X] Update `CHANGELOG.rst` 
- [x] Update `pollbot/api.yaml`
- [x] Create your task in `pollbot/tasks`
- [x] Make sure a heartbeat exists for this service
- [x] Create the view in `pollbot/views/release.py`
- [x] Add the new endpoints in `pollbot/app.py`
- [x] Fix tests, flake8 and coverage

### CODE REVIEW

File: pollbot/api.yaml [hunk 1]
```diff
@@ -259,6 +259,23 @@ paths:
       tags:
       - Status
 
+  /{product}/{version}/crash-stats/uptake:
+    get:
+      summary: "checks what is the version uptake with crash-stats info."
```
Comment (glasserc) [inline pollbot/api.yaml]: Do we put periods at the end of these?

File: pollbot/tasks/crash_stats.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    channel = get_version_channel(version)
+    url = '{}/ProductVersions/?active=true&build_type={}&product={}'.format(
+        CRASH_STATS_SERVER, channel.value, product)
+    with get_session() as session:
+        async with session.get(url) as resp:
+            body = await resp.json()
+            versions = [h['version'] for h in body['hits'][:5]]
+            if version not in versions:
+                versions.extend([h['version'] for h in body['hits'] if h['version'] == version])
+            return versions
```
Comment (glasserc) [inline pollbot/tasks/crash_stats.py]: A comment or an issue would be super helpful to explain what is going on here. If I'm reading this right, we want to get the first five versions from CrashStats, but we also want to add the version we asked about if it's available anywhere in the list? Why do we need both the first five and the one we asked about?

File: pollbot/tasks/crash_stats.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    if channel is Channel.BETA:
+        versions = ['{}b'.format(version.split('b')[0])]
+    else:
+        versions = await get_channel_versions(product, version)
+
+    url = ('{}/ADI/?start_date={}&end_date={}&'
+           'platforms=Windows&platforms=Linux&platforms=Mac%20OS%20X&'
+           'product={}&versions={}')
+    url = url.format(CRASH_STATS_SERVER, date, date, product,
+                     '&versions='.join(versions))
```
Comment (glasserc) [inline pollbot/tasks/crash_stats.py]: This `join` is clever!

File: pollbot/views/utilities.py [hunk 4]
```diff
@@ -45,14 +45,16 @@ def render_yaml_file(filename):
...
     return web.json_response({"archive": info[0],
-                              "balrog": info[1],
-                              "bedrock": info[2],
-                              "buildhub": info[3],
-                              "product-details": info[4]},
+                              "crash-stats": info[1],
+                              "balrog": info[2],
+                              "bedrock": info[3],
+                              "buildhub": info[4],
+                              "product-details": info[5]},
```
Comment (glasserc) [inline pollbot/views/utilities.py]: Why add crash-stats where you did? I guess I thought this was alphabetical order, but it came before `balrog`.

File: tests/test_views.py [hunk 5]
```diff
@@ -286,7 +292,8 @@ def cli(loop, test_client):
     }, status=404)
 
 
-# This is currently a functional test.
+# This are currently functional tests.
```
Comment (glasserc) [inline tests/test_views.py]: `These are`

File: tests/test_views.py [hunk 6]
```diff
@@ -336,14 +353,28 @@ def cli(loop, test_client):
...
+    assert body['link'].startswith("https://crash-stats.mozilla.com/api/ADI/")
+    assert body['message'].startswith("Crash-Stats uptake for version 54.0 is")
+
+
+async def test_beta_crash_stats_uptake(cli):
+    resp = await check_response(cli, "/v1/firefox/56.0b10/crash-stats/uptake")
+    body = await resp.json()
+    assert body['status'] == Status.INCOMPLETE.value
+    assert body['link'].startswith("https://crash-stats.mozilla.com/api/ADI/")
+    assert body['message'].startswith("Crash-Stats uptake for version 56.0b10 is")
```
Comment (glasserc) [inline tests/test_views.py]: Are we sure this won't ever become complete?

File: pollbot/tasks/crash_stats.py (hunk 2 above)
Comment (Natim) [reply pollbot/tasks/crash_stats.py]: It is just in case the one we want is not part of the 5 last ones.

File: pollbot/views/utilities.py (hunk 4 above)
Comment (Natim) [reply pollbot/views/utilities.py]: Yes iti is alphabetical and I messed up with b and c order 😢

File: tests/test_views.py (hunk 6 above)
Comment (Natim) [reply tests/test_views.py]: Not for past release, it can only getting worst.

**Pull request merged**